app/controllers/subscriptions_controller.rb

```diff
@@ -28,6 +28,20 @@ def create
     render json: { id: subscription.id }, status: status
   end
 
+  def index
+    subscriber = find_subscriber
+
+    return render_not_found if subscriber.nil?
+
+    subscriptions = Subscription.active.
+      joins(:subscriber_list).
+      where(subscriber: subscriber).
+      order('subscriber_lists.title').
+      select(:id, :'subscriber_lists.title', :frequency).as_json
```

Comment:
I wonder if we should use a `to_json` method on `Subscription` rather than selecting the fields like we do with `SubscriberList`.

Reply:
I've changed it so that we return the full subscriber and subscription details